tests/optim/test_oss.py

```diff
@@ -619,48 +678,36 @@ def run_state_dict_distributed(rank, world_size, tempfile_name):
     sharded_optimizer2 = optim.OSS(model_oss2.parameters(), lr=0.1, momentum=0.99)
     sharded_optimizer2.add_param_group({"params": head_oss2.parameters()})
```

Comment:
@msbaines this uses multiple param groups (through the  `.add_param_groups` interface that I mentioned another time)